src/modules/my-positions/selectors/my-positions.js

```diff
@@ -11,6 +10,5 @@ export default function () {
 export const selectPositionsMarkets = createSelector(
   selectMarkets,
   selectAccountPositionsState,
-  selectAllUserOpenOrderMarkets,
-  (markets, positions, marketIDs) => (markets || []).filter(market => Object.keys(positions || {}).find(positionMarketID => market.id === positionMarketID) || (marketIDs || {}).find(marketID => market.id === marketID))
+  (markets, positions) => (markets || []).filter(market => Object.keys(positions || {}).find(positionMarketID => market.id === positionMarketID))
```

Comment:
What is the intended functionality of this selector?  I think it's not working the way you expect it to, since the filter function's return value isn't Boolean.  If you just want to pull out an array of market objects in which the user has positions, I think all you need is `(markets || []).filter(market => (positions || {})[market.id] != null)`, although honestly I'd break this apart into 2 lines for readability and efficiency's sake:
```javascript
(markets, positions) => {
  if (!markets || !positions) return []
  return markets.filter(market => positions[market.id] != null)
}
```

Reply:
I agree, I got confirmation form @stephensprinkle that having positions and orders together is left over when data was being pulled from contracts. 

I'll simplify it, can delete this selector and do it single line in caller.